VAMobile/src/components/CommonErrorComponents/DowntimeError.tsx

```diff
@@ -31,9 +29,12 @@ const DowntimeError: FC<DowntimeErrorProps> = ({ screenID, overrideFeatureName }
     mb: theme.dimensions.contentMarginBottom,
   }
   const { downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  const feature = DowntimeScreenIDToFeature[screenID]
-  const featureName = overrideFeatureName ? overrideFeatureName : downtimeWindowsByFeature[feature]?.featureName
-  const endTime = downtimeWindowsByFeature[feature]?.endTime.toFormat('fff')
+  const feature = ScreenIDToDowntimeFeature[screenID]
+  const downtimeWindow = feature ? downtimeWindowsByFeature[feature] : null
+
+  const featureName = ScreenNameFeatureConstants[screenID]
+  // const featureName = t(`downtime.${screenID}.featureName`)
+  const endTime = downtimeWindow?.endTime.toFormat('fff')
```

Comment:
This is one of the reasons I didn't bother making `ScreenIDToDowntimeFeature` 1:many. If we did that, we'd need to define rules for how it's handled when multiple are down. Which endtime does it use?